app/src/main/java/com/mobileproto/hireddit/hireddit/visuals/SpeechFragment.java

```diff
@@ -130,6 +130,7 @@ public void doListen() {
         Log.d(DEBUG_TAG, "Start listening");
         isListening = true;
         updateListeningIndicator();
+        mListener.speak("");
```

Comment:
I don't know which way is better, because Main activity speech object is not static, so it currently cannot be called from other than the life cycle events. We can of course make speech object static in the main activity and it will work. But if this way works, we don't have to make it static.